etna/records/models.py

```diff
@@ -182,12 +182,20 @@ def hierarchy_level3_summary_title(self) -> str:
     @cached_property
     def next_record(self) -> Union[Dict[str, str], None]:
         if record := self.get("@next"):
-            return {"iaid": extract(record, "@admin.id")}
+            ref_num = self.get("identifier.0.reference_number")
```

Comment:
@jamesbiggs1 I don't think `self.get()` is the right thing to use here... that will give you the current record's reference number, when what you want is the one for the next record.